document_page_approval/models/document_page.py

```diff
@@ -122,14 +122,17 @@ def can_user_approve_this_page(self, user):
         # if it's not required, anyone can approve
         if not self.is_approval_required:
             return True
-        # to approve, you must have approver rights
-        approver_group_id = self.env.ref(
-            'document_page_approval.group_document_approver_user')
-        if approver_group_id not in user.groups_id:
+        # if user belongs to 'Knowledge / Manager', he can approve anything
+        if user.has_group('document_page.group_document_manager'):
+            return True
```

Comment:
This is actually an improvement, not a FIX.
Also rewrited the code to use `has_group` instead of manually checking for `groups_id`